### PR TITLE
Sql 'is' operator added

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by tejainece** [comment]
Hi,

Thanks for the PR. I see that you are using `iq` for the operator. What do you think about using `iss`? I feel `iss` is bit more intuitive.

@jaumard

**Comment by ghost** [comment]
Hi, Ok, I agree=)

File: query/lib/src/core/expression/field.dart [hunk 1]
```diff
@@ -27,6 +27,13 @@ class Field<ValType> {
   ///     find.where(age.eq(20));
   Cond<ValType> eq(ValType value) => Cond.eq<ValType>(this, value);
 
+  /// Returns an "is" condition, i.e. 'where var IS null'
+  ///
+  ///     FindStatement find = new FindStatement();
+  ///     Field<int> author = new Field<int>('age');
+  ///     find.where(age.eq(null));
```
Comment (jaumard) [inline query/lib/src/core/expression/field.dart]: I guess the example is wrong here it use `eq`

**Comment by jaumard** [comment]
Ok for `iss` too, more intuitive than `iq`

File: query/lib/src/core/expression/field.dart (hunk 1 above)
Comment (ghost) [reply query/lib/src/core/expression/field.dart]: You can do everything as you wish, the main thing I need is the method I added =)

Comment (jaumard) [reply query/lib/src/core/expression/field.dart]: No problem with the new method, but the doc should be correct or removed  ;)

Comment (ghost) [reply query/lib/src/core/expression/field.dart]: If you think it is better to do differently, you can do, I agree to this

**Comment by ghost** [comment]
I changed the method name and comments.

**Review by tejainece — APPROVED** [review]
LGTM

**Comment by tejainece** [comment]
@gold-kuler Thanks for the PR.

@jaumard Can you implement the operator for sqlite? I will implement it for postgres.

**Comment by hjJunior** [comment]
@tejainece and @jaumard I have open a PR to implement for sqflite

**Comment by jaumard** [comment]
Just think that it will be nice to also have `is not` operator too ^^

**Comment by hjJunior** [comment]
@jaumard maybe I work on `is not` operator, do u mind?

**Comment by jaumard** [comment]
PR is already opened @hjJunior :) as I had to implement alter for sqflite it didn't take long to add it too

**Comment by tejainece** [comment]
@jaumard Published!

**Pull request merged**